upload/catalog/controller/extension/payment/coinbase.php

```diff
@@ -157,7 +158,10 @@ public function updateRecord($data){
                 $status_message = 'Coinbase Commerce Status ' . $data['coinbaseStatus'] . ' Type ' . $data['type'];
 
                 if ($data['coinbaseStatus'] == 'NEW' && $data['type'] == 'charge:created') {
-                    $order_status = 'coinbase_created_status_id';  //Pending
+                    $order_status = 'coinbase_created_status_id';  //Created
+                } elseif ($data['coinbaseStatus'] == 'PENDING' && $data['type'] == 'charge:pending') {
+                    $order_status = 'payment_coinbase_pending_status_id';  //Pending

```

Comment:
does this handle cancelled?

Reply:
Yes 
upload/catalog/controller/extension/payment/coinbase.php

            "redirect_url" => $this->url->link('extension/payment/coinbase/redirect&orderId=' . $order_info["order_id"], true),
            "cancel_url" => $this->url->link('checkout/checkout')

Reply:
Cool, assuming this is still in your local branch?

Reply:
![image](https://user-images.githubusercontent.com/1286265/48081291-56406580-e1f8-11e8-97c9-a3d22e328b42.png)
Current PR.